app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt

```diff
@@ -152,7 +153,7 @@ fun FlightScreen(
                 )
 
             inFlightViewModel.addLocation(Location(userId = uid, point = newLocation))
-
+            inFlightViewModel.saveFlightTrace()

```

Comment:
You shouldn't save the flight trace here (this will be handled in #321)

Reply:
Didn't know you were going to manage that